generate-blog-posts.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs')
+const data = require('./src/data/blog/posts.json')
+
+const { posts } = data.db[0].data
+
+// Converts "null" to null, "false" to false etc. Everything is returned as a string wrapped in "s
+const convertType = value => {
+  try {
+    return new Function('return ' + value + ';')()
```

Comment:
So this feels `eval`. Can't we use JSON.parse instead?
```js
JSON.parse('null') === null
JSON.parse('true') === true
```
```suggestion
    return JSON.parse(value)
```